src/grilodatasource.cpp

```diff
@@ -446,3 +446,18 @@ void GriloDataSource::timerEvent(QTimerEvent *event)
         Q_EMIT contentUpdated();
     }
 }
+
+guint GriloDataSource::getOpId() const
+{
+    return m_opId;
+}
+
+void GriloDataSource::setOpId(guint id)
+{
+    m_opId = id;
+}
+
+GriloRegistry * GriloDataSource::getGriloRegistry() const
```

Comment:
nit: should not have space between '*' and var/func. ditto for declaration in header